sdks/wasm/driver.c

```diff
@@ -214,6 +222,28 @@ mono_wasm_invoke_js (MonoString *str, int *is_exception)
 #include "driver-gen.c"
 #endif
 
+struct WasmAssembly_ {
+	MonoBundledAssembly assembly;
+	struct WasmAssembly_ *next;
+};
+
+typedef struct WasmAssembly_ WasmAssembly;
```

Comment:
If move typedef to before struct definition, then can say `WasmAssembly *next` instead of `struct WasmAssembly_ *next`.